### PR TITLE
simplify univalence proof

### PR DESCRIPTION
**Pull request by ecavallo**
down with connections!!

### CODE REVIEW

**Comment by jonsterling** [comment]
cool!

**Comment by ecavallo** [comment]
The indentation under `, λ` was/is not consistent in here, but I forget which one is JST.

**Comment by jonsterling** [comment]
> The indentation under `, λ` was/is not consistent in here, but I forget which one is JST.

JST is to not indent underneath lambda, or anything that binds everytihng to its right.

**Comment by jonsterling** [comment]
I was fiddling with this file and I'm noticing a LOT of mortality in either the elaborator or the typechecker concerning `V` types. I hope at least some of it is resolved by my new type checker.

File: library/prelude/univalence.red [hunk 1]
```diff
@@ -3,41 +3,47 @@ import prelude.connection
 import prelude.hlevel
 import prelude.equivalence
 
-def ua (A B : type) (E : equiv A B) : path^1 type A B =
-  λ i →
-    `(V i A B E)
+def ua (A B : type) (e : equiv A B) : path^1 type A B =
+  λ i → V i A B e
```
Comment (favonia) [inline library/prelude/univalence.red]: I thought I already updated this LOL!

**Pull request merged**